applications/filestore-backup/templates/cronjobs.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: create-backup
+spec:
+  schedule: {{ .Values.tool.backup.schedule | quote }}
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          automountServiceAccountToken: true
+          {{- with .Values.tolerations }}
+          tolerations:
+{{ toYaml . | indent 12 }}
+          {{- end }}
+          {{- with .Values.tolerations }}
+          affinity:
+{{ toYaml . | indent 12 }}
+          {{- end }}
+          containers:
+          - name: create-backup
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+            securityContext:
+              allowPrivilegeEscalation: false
+              runAsNonRoot: true
+              runAsUser: 1000
+              runAsGroup: 1000
+              capabilities:
+                drop:
+                  - all
+              readOnlyRootFilesystem: true
+            env:
+              - name: "RUBIN_GOOGLE_FILESTORE_TOOLS_PROJECT"
+                value: {{ .Values.global.gcpProjectId | quote }}
+              - name: "RUBIN_GOOGLE_FILESTORE_TOOLS_ZONE"
+                value: "{{ .Values.global.gcpRegion }}-{{ .Values.tool.zone }}"
+              - name: "RUBIN_GOOGLE_FILESTORE_TOOLS_INSTANCE"
+                value: {{ .Values.tool.instance | quote }}
+              - name: "RUBIN_GOOGLE_FILESTORE_TOOLS_SHARE_NAME"
+                value: {{ .Values.tool.fileShare | quote }}
+              {{- with .Values.tool.backup.debug }}
+              - name: "RUBIN_GOOGLE_FILESTORE_TOOLS_DEBUG"
+                value: "true"
+              {{- end }}
+            command: [ "create_backup" ]
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: purge-backup
```

Comment:
Normal metadata is missing.

I'm not really a fan of putting multiple resources in the same file unless they make no sense at all in isolation. I know we've done this in a few places, but for resources this long, I think it would make them more readable to break them into two files, one to create the backups and one to purge them.